docs/configuration.md

```
@@ -0,0 +1,57 @@
+# Configuration Guide
+
+## Environment Variables
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)